pkg/install/install.go

```diff
@@ -56,3 +56,11 @@ func PrepareLokomotiveTerraformModuleAt(path string) error {
 	}
 	return nil
 }
+
+func PrepareChartDirAt(path string) error {
```

Comment:
Why not just have a generic function to extract the directory from assets to disk?

Reply:
No reason at all, I copied the existing behavior (e.g. `PrepareLokomotiveTerraformModuleAt()`) to keep the scope of this PR small. I'm happy to merge all of the "Prepare*" functions to a single one.

Reply:
Fixed. I took this opportunity to refactor our assets handling. Details and rationale are in the commit messages.

Reply:
Cool, I'll have a look!